docs/cmdline-opts/write-out.d

```diff
@@ -115,6 +115,14 @@ From this point on, the --write-out output will be written to standard output.
 This is the default, but can be used to switch back after switching to stderr.
 (Added in 7.63.0)
 .TP
+.B content
+From this point on, the result of response content.
```

Comment:
I will update this and demonstrates more details.
>(and perhaps: can we come up with a way that can offer the raw data as well as an escaped version?)
Yes, of course. If user not add quotes for %{content}. It will show the raw data otherwise display the escaped string.

Reply:
```txt
.TP
.B response_body
The response body that the resouce requested by curl. It will be escaped by JSON
format if you add quotes for it.(eg. "%{response_body}")
(Added in 7.66.0)
.TP
.B error_message
The error message that curl returned. It will be escaped by JSON format if you
add quotes for it.(eg. "%{error_message}")
(Added in 7.66.0)

```
@bagder  Do u think this is OK?

Reply:
I think you should leave out the escaping part entirely in the first step and then provide a generic way to output escaped variables for write-out. I think it would be better to do that using a more explicit way than if the variable happens to have double-quotes around it. (since some users will prefer single-quotes or not using quotes at all etc).

Perhaps it could be an instruction within the braces?

    %{variable:q}

Where `q` stands for "quoted". (This would also allow for other instructions if someone has other ideas down down the line...)

Reply:
OK, @bagder  I will adopt ur approach.